plugins/headlamp-plugin/template/.vscode/settings.json

```diff
@@ -0,0 +1,26 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "files.exclude": {
+    "build": false // set this to true to hide the "dist" folder with the compiled JS files
+  },
+  "search.exclude": {
+    "build": true // set this to false to include "dist" folder in search results
+  },
+  // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
+  // "typescript.tsc.autoDetect": "off",
```

Comment:
Should this setting be commented out?

Reply:
I've removed it. Doesn't seem to be the case that we need it as off. I wasn't sure before.

Reply:
It's still here, right?

Reply:
It's removed now. (Seems I didn't push before. )